spec/controllers/openid_connect/authorization_controller_spec.rb

```diff
@@ -319,7 +319,7 @@
                 it 'redirects to the redirect_uri immediately when pii is unlocked if client-side redirect is disabled' do
```

Comment:
Should we update the description to match what we're now asserting?

Reply:
Good idea, updated.